tests/wallet/sync/test_wallet_sync.py

```diff
@@ -1184,11 +1185,17 @@ async def new_func(*args, **kwargs):
 
             return new_func
 
-        def flaky_fetch_coin_spend(node, func):
+        request_puzzle_solution_failure_tested = False
+
+        def flaky_request_puzzle_solution(func):
```

Comment:
renaming this test doesn't look aligned with the other renames of `puzzle_solution` -> `coin_spend` renames

Reply:
Its not really renamed in the final diff unless i miss something? And it tests the flakiness of `request_puzzle_solution` on the fullnode side so i think the naming is fine?